### PR TITLE
Add clickable web link support

### PR DESCRIPTION
**Pull request by Pierre977**
Add clickable web link support to SlideFragment's title and text. Now resource string could contain <a> tag and it will be clickable.

### CODE REVIEW

**Comment by Pierre977** [comment]
Sorry, I do not understand the reason of the conflict, because I just added two attributes to the textviews. 
android:linksClickable="true"
        android:autoLink="web"

**Comment by HeinrichReimer** [comment]
You are probably checking our from an old commit.

File: library/src/main/res/layout/fragment_simple_slide.xml [hunk 1]
```diff
@@ -27,6 +27,8 @@
         android:gravity="center"
         android:textAppearance="@style/TextAppearance.AppCompat.Headline"
         android:fontFamily="sans-serif-medium"
+        android:linksClickable="true"
```
Comment (HeinrichReimer) [inline library/src/main/res/layout/fragment_simple_slide.xml]: Missing the attributes for the second text view

**Comment by Pierre977** [comment]
I have recreate the fork and re-do the pull request. Thanks for your answers. I will close this pull request.

**Comment by HeinrichReimer** [comment]
Fixed via #192

**Pull request closed without merging**